src/Services/DeviceManager.vala

```diff
@@ -117,6 +118,15 @@ public class Power.Services.DeviceManager : Object {
         });
 
         has_battery = batteries.has_next ();
+
+        // Only interested in devices if they're discharging peripherals or supplying power to the computer
+        only_charging_peripheral_devices = true;
+        foreach (var device in devices.values) {
+            if (device.power_supply || device.state == DEVICE_STATE_DISCHARGING || device.state == DEVICE_STATE_EMPTY) {
```

Comment:
```suggestion
            if (device.power_supply || device.state == DISCHARGING || device.state == EMPTY) {
```

To reflect changes in [this commit](https://github.com/elementary/wingpanel-indicator-power/commit/aa920e13acbab3292b7e0422018d58ab1d380154#diff-e522e471998bdef45f66d6adffe919ad)